Transcom/Parsers/TimeTableParser.cs

```diff
@@ -0,0 +1,61 @@
+﻿using PSITranscom.Models;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Transcom.Exceptions;
+using Transcom.Factories.ScheduleFactory;
+using static Transcom.Constants;
+
+namespace Transcom.Parsers
+{
+    public class TimetableParser<T> : IParser<Timetable> where T : Timetable
```

Comment:
No need for the class to be generic when you're not using the generic type parameter
```suggestion
    public class TimetableParser : IParser<Timetable>
```